package.json

```diff
@@ -8,28 +8,24 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "create-player-link": "cd dist/quml-library && npm link && cd ../..",
```

Comment:
@itsvick are we not using this?

Reply:
Yes, we've stopped using this and added same in the tsconfig file, so that it will pick the changes from the dist folder itself.
@pallakartheekreddy